packages/core/types/config.py

```diff
@@ -21,28 +21,28 @@ class StricterBaseModel(pydantic.BaseModel):
 class StrictFilePath(pydantic.RootModel[str]):
```

Comment:
There are a lot of formatting changes in this file still.

Reply:
No idea. That is what the formatting with YAPF put out.

Reply:
Ok great. Then maybe it was not formatted for a long time before ^^